core/save-git.c

```diff
@@ -431,7 +431,8 @@ static void create_dive_buffer(struct dive *dive, struct membuffer *b)
 	SAVE("visibility", visibility);
 	cond_put_format(dive->tripflag == NO_TRIP, b, "notrip\n");
 	save_tags(b, dive->tag_list);
-	cond_put_format(!!dive->dive_site, b, "divesiteid %08x\n", dive->dive_site->uuid);
+	if (dive->dive_site)
+		cond_put_format(!!dive->dive_site, b, "divesiteid %08x\n", dive->dive_site->uuid);
```

Comment:
Super-minor nitpick: this can now be replaced by a `put_format`, since the condition is in the line above.

Reply:
yeah, this should be just a `put_format()`